src/components/notification-center/NotificationCenter/styles.module.css

```diff
@@ -1,7 +1,7 @@
 .bell {
   display: flex;
   justify-content: center;
-  height: 100%;
+  padding: 4px;
```

Comment:
Is the padding just to increase the clickable area? maybe we could make the whole square clickable?

Reply:
After adding the focus style I noticed that the area looked strange so I removed the full height and added a consistent padding. We could also change it to take up the full size but then I would suggest to do it for all header elements.

Before/After:
<img width="62" alt="Screenshot 2023-01-03 at 11 33 07" src="https://user-images.githubusercontent.com/5880855/210340487-b248678e-76bb-429e-8e57-a398e1a098ab.png">
<img width="73" alt="Screenshot 2023-01-03 at 11 32 59" src="https://user-images.githubusercontent.com/5880855/210340513-9cb03547-14d4-4e1d-bbf7-a97c2f5867fd.png">

Reply:
Yeah, I think we can leave it per your change. I'm going to resolve this, it is non-blocking.